sigstore/_internal/tuf.py

```diff
@@ -156,7 +157,12 @@ def _get(self, usage: str, statuses: list[str]) -> list[bytes]:
                 if path is None:
                     path = self._updater.download_target(target_info)
                 with open(path, "rb") as f:
-                    data.append(f.read())
+                    target_cert = f.read()
+                    logger.debug(
+                        f"TUF cache target {usage} {statuses}: {os.path.basename(path)} with hash\n"
+                        f"{target_cert.decode('utf-8')}"
+                    )
+                    data.append(target_cert)
```

Comment:
These can really be anything (certs, bare public keys, etc.), so I'm going to rename it to `target_contents`.

```suggestion
                    target_contents = f.read()
                    logger.debug(
                        f"TUF cache target {usage} {statuses}: {os.path.basename(path)} with hash\n"
                        f"{target_contents.decode('utf-8')}"
                    )
                    data.append(target_contents)
```